README.md

```diff
@@ -206,6 +204,49 @@ defmodule AnotherService.GetHats do
 end
 ```
 
+### Running with Phoenix
+
+The plug can also be attached within a Phoenix Router. Example below would be accessible at `/rpc/hat` 
+
+URL: `/rpc/hat/twirp/example.Haberdasher.MakeHat` or `/{prefix?}/twirp/{package}.{service}/{method}`
+
+```elixir
+defmodule ExampleWeb.Router do
+  use FlytroHealthWeb, :router
```

Comment:
We can probably update this to ExampleWeb ;)

Reply:
Done and thanks for the review @keathley